src/components/app-bar/components/nav-menu-button/index.tsx

```diff
@@ -0,0 +1,2 @@
+export * from './nav-menu-button'
```

Comment:
should we use `app-bar-menu-button` instead though? since the term `nav` probably refer to older component 🤔